dynamic_rest/content_type_field.py

```diff
@@ -0,0 +1,73 @@
+import traceback
```

Comment:
Packaging: since we already have serializer fields avaialble at `dynamic_rest.fields`, we may want to turn `fields` into a directory and make this class live at `dynamic_rest.fields.content_type:DynamicContentTypeField`